include/z64camera.h

```diff
@@ -15,15 +15,20 @@
 #define CAM_STAT_UNK100     0x100
 
 #define NUM_CAMS 4
-#define MAIN_CAM 0
-#define SUBCAM_FIRST 1
-#define SUBCAM_FREE 0
-#define SUBCAM_NONE -1
-#define SUBCAM_ACTIVE -1
+
+// Camera IDs are indices into `GlobalContext.cameraPtrs`
+// The main camera is at index 0
+#define CAM_ID_MAIN 0
+// The sub cameras start at index 1
+#define CAM_ID_SUB_FIRST 1
+// -1 is used to indicate "no camera"
+#define CAM_ID_NONE -1
+// 0 is also used for variables holding sub camera IDs to indicate "no camera"
+#define SUB_CAM_ID_DONE 0
```

Comment:
```suggestion
// Camera IDs are indices into `GlobalContext.cameraPtrs`
#define CAM_ID_MAIN 0 // The main camera is at index 0
#define CAM_ID_SUB_FIRST 1 // The sub cameras start at index 1
#define CAM_ID_NONE -1 // -1 is used to indicate "no camera"

#define SUB_CAM_ID_DONE 0 // In some actors, 0 is also used for variables holding sub camera IDs to indicate "subcam is finished"
```
just looks neater this way imo. And I added "in some actors" to the last define, cause afaik this doesnt mean anything to camera/z_play and is just a local meaning to some actors

Reply:
also can probably afford to take the values themselves out of the comments. you can look at the definition to see what the number is

Reply:
for example
```
#define CAM_ID_MAIN 0 // The index of the main camera
#define CAM_ID_SUB_FIRST 1 // The index sub cameras start at
#define CAM_ID_NONE -1 // Used to indicate no camera. Can be used to default to the active camera in some scenarios

#define SUB_CAM_ID_DONE 0 // Used in some actors for variables holding sub camera IDs to indicate "subcam is finished"

Reply:
I didnt mean to remove // Camera IDs are indices into `GlobalContext.cameraPtrs` at the top, it probably still helps give context for what these groups of defines are for